src/content/docs/tutorials/counter-app/basic-app.md

```diff
@@ -60,7 +60,7 @@ A common pattern found in most Ratatui apps is that they:
 3. Restore the terminal back to its original state
 
 The `main` function sets up the terminal by calling the `ratatui::init` and `ratatui::restore`
-methods and then creates and runs the App (defined later). It defers evaluating the result of
+methods and then creates and runs the App (defined later). It defers the return of the result from
```

Comment:
Maybe using "defers the result of calling App::run" would be more simple & clear here?

Reply:
I think this is related to the subject; it makes more sense that the subject of "defer" is "return" than `result`.

Reply:
The rust book specifically uses the word "propagating", which is what was intended by this.

https://doc.rust-lang.org/book/ch09-02-recoverable-errors-with-result.html#propagating-errors

Reply:
It's not about "deferring evaluation" - the App::run() method is actually executed immediately.